LibTest/js_interop/JS/js_A04_t02.dart

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion An annotation on a JavaScript interop declaration.
+/// ...
+/// Specifying name customizes the JavaScript name to use, which can be used in
+/// the following scenarios:
+/// - Adding a JavaScript prefix to all the external top-level declarations,
+///   static members, and constructors of a library by parameterizing the
+///   annotation on the library with `name`.
+/// - Specifying the JavaScript class to use for external static members and
+///   constructors of an interop extension type by parameterizing the annotation
+///   on the interop extension type with `name`.
+/// - Renaming external declarations by parameterizing the annotation on the
+///   member with `name`.
+///
+/// @description Check that `@JS(null)` is equivalent `@JS()`.
```

Comment:
Typo:
```suggestion
/// @description Check that `@JS(null)` is equivalent to `@JS()`.
```

Reply:
This would probably be a necessity, because `JS()` would accept a positional parameter optionally and with default value null. So that makes sense and should not be an error nor a warning.